build.sbt

```diff
@@ -227,9 +230,8 @@ lazy val storehausMemcache = module("memcache").settings(
     "com.twitter" %% "bijection-core" % bijectionVersion,
     "com.twitter" %% "bijection-netty" % bijectionVersion,
     "com.twitter" %% "finagle-memcached" % finagleVersion excludeAll(
-      // we don't use this and its not on maven central.
-      ExclusionRule("com.twitter.common.zookeeper"),
-      ExclusionRule("com.twitter.common")
```

Comment:
Part of the upgraded finagle memcache client needed com.twitter.common.metrics.Gauge so this exclude was removed.